### PR TITLE
Implement the WES equivalent of metadata

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: engine/src/test/scala/cromwell/webservice/routes/wes/WesRouteSupportSpec.scala [hunk 1]
```diff
@@ -149,3 +172,41 @@ class WesRouteSupportSpec extends AsyncFlatSpec with ScalatestRouteTest with Mat
...
+              |      "stderr": "/Users/jgentry/projects/cromwell/cromwell-executions/supsup/cab8b246-8c78-4ca9-b2c3-1c8377da986e/call-sup/execution/stderr"
+              |    }
+              |  ],
+              |  "run_log": {
+              |    "name": "supsup",
+              |    "start_time": "2018-12-14T14:25:08.124-05:00",
+              |    "end_time": "2018-12-14T14:25:25.703-05:00"
+              |  }
+              |}""".stripMargin.parseJson.asInstanceOf[JsObject]
+}
```
Comment (mcovarr) [inline engine/src/test/scala/cromwell/webservice/routes/wes/WesRouteSupportSpec.scala]: \n

File: engine/src/main/scala/cromwell/webservice/routes/wes/CromwellMetadata.scala [hunk 2]
```diff
@@ -0,0 +1,110 @@
+package cromwell.webservice.routes.wes
+
+import cromwell.core.WorkflowState
+import spray.json.{DefaultJsonProtocol, JsObject, JsonFormat, JsonParser}
+
+/**
+  * As described in WesRouteSupport the least bad solution to reshaping the outgoing metadata JSON at the moment
+  * is to request Cromwell's metadata JSON, parsing it back into Scala objects, and then outputting back into
+  * the correct WES JSON.
```
[the remark this answers is not in the document]
Comment (geoffjentry) [reply engine/src/main/scala/cromwell/webservice/routes/wes/CromwellMetadata.scala]: What do you mean?

[the remark this answers is not in the document]
Comment (geoffjentry) [reply engine/src/main/scala/cromwell/webservice/routes/wes/CromwellMetadata.scala]: Discussed on slack. Going to take a look as it seems to be just what the doctor ordered

**Comment by geoffjentry** [comment]
Closing until I have a chance to learn me some jq for great good

**Pull request closed without merging**